app/file.go

```diff
@@ -673,6 +673,20 @@ func (t *uploadFileTask) runPlugins() *model.AppError {
 }
 
 func (t *uploadFileTask) preprocessImage() *model.AppError {
+	// If SVG, attempt to extract dimensions and then return
+	if t.fileinfo.MimeType == "image/svg+xml" {
+		svgInfo, err := parseSVG(t.newReader())
+		if err != nil {
+			mlog.Error(fmt.Sprintf("Unable to parse SVG, err = %v", err))
```

Comment:
@deanwhillier, let's write this as a structured log:
```suggestion
			mlog.Error("Failed to parse SVG", mlog.Err(err))
```